novuRajan/gannt-chart
# Today date

## Description of the problem this PR solves
Today date was not highlighted.
we should be able to highlight the important dates as well.
## Details of how the solution has been implemented
check the today date. if it is today date then it will be highlighted with red color and same goes for the important dates as well
## Testing Instructions
npm run dev

```
 GanttChart.createGanttChart(data,{
        containerId: 'chart',
        importantDates: [
            {
                date: '2023-10-04',
                label: 'Start Date',
            },
            {
                date: '2023-10-18',
                label: 'End Date',
            }
        ],
    });
```

insert this code in HTML file you will see the important date highlighted with red color and show its name as well

## Browsers tested
Select all browsers the solution was tested on <br>
*(test on as many as possible if the solution is front-end, otherwise this section is optional)*

## Notes
Any further information required for this PR to be reviewed and tested
